src/scss/skin-modern/_skin.scss

```diff
@@ -1,94 +1,93 @@
+@import 'components/titlebar';
+@import 'components/recommendationoverlay';
+@import 'components/clickoverlay';
+@import 'components/hugereplaybutton';
+@import 'components/playbacktimelabel';
+@import 'components/bufferingoverlay';
+@import 'components/playbacktoggleoverlay';
+@import 'components/closebutton';
+@import 'components/airplaytogglebutton';
+@import 'components/pictureinpicturetogglebutton';
+@import 'components/spacer';
+@import 'components/subtitlesettings/subtitlesettings';
+@import 'components/subtitlesettingspaneltogglebutton';
+@import 'components/audiotracksettingspaneltogglebutton';
+@import 'skin-ads';
+@import 'skin-cast-receiver';
+@import 'skin-smallscreen';
 
-.#{$prefix}-ui-skin-modern {
-  @import 'mixins';
-  @import 'components/component';
-  @import 'components/container';
-  @import 'components/uicontainer';
-  @import 'components/controlbar';
-  @import 'components/button';
-  @import 'components/playbacktogglebutton';
-  @import 'components/fullscreentogglebutton';
-  @import 'components/vrtogglebutton';
-  @import 'components/volumetogglebutton';
-  @import 'components/seekbar';
-  @import 'components/watermark';
-  @import 'components/hugeplaybacktogglebutton';
-  @import 'components/label';
-  @import 'components/settingspanel';
-  @import 'components/settingspanelpage';
-  @import 'components/settingspanelpageopenbutton';
-  @import 'components/settingspanelpagebackbutton';
-  @import 'components/settingstogglebutton';
-  @import 'components/selectbox';
-  @import 'components/listbox';
-  @import 'components/seekbarlabel';
-  @import 'components/volumeslider';
-  @import 'components/subtitleoverlay';
-  @import 'components/subtitleoverlay-cea608';
-  @import 'components/volumecontrolbutton';
-  @import 'components/casttogglebutton';
-  @import 'components/caststatusoverlay';
-  @import 'components/errormessageoverlay';
-  @import 'components/titlebar';
-  @import 'components/recommendationoverlay';
-  @import 'components/clickoverlay';
-  @import 'components/hugereplaybutton';
-  @import 'components/playbacktimelabel';
-  @import 'components/bufferingoverlay';
-  @import 'components/playbacktoggleoverlay';
-  @import 'components/closebutton';
-  @import 'components/airplaytogglebutton';
-  @import 'components/pictureinpicturetogglebutton';
-  @import 'components/spacer';
-  @import 'components/subtitlesettings/subtitlesettings';
-  @import 'components/subtitlesettingspaneltogglebutton';
-  @import 'components/audiotracksettingspaneltogglebutton';
-  @import 'skin-ads';
-  @import 'skin-cast-receiver';
-  @import 'skin-smallscreen';
-
+.#{$prefix}-ui-uicontainer {
   color: $color-primary;
   font-family: $font-family;
   font-size: $font-size;
   text-align: left;
   user-select: none;
+}
 
-  &.#{$prefix}-player-state-idle {
-    .#{$prefix}-ui-controlbar,
-    .#{$prefix}-ui-titlebar,
-    .#{$prefix}-ui-hugeplaybacktogglebutton {
-      display: none;
-    }
+&.#{$prefix}-player-state-idle {
```

Comment:
I think you need to nest all into `.#{$prefix}-ui-skin-modern {`, else the `&` extender does not really work as there is no parent?

Reply:
FYI: It actually behaves like the `&` isn't there. 

It looks a bit weird so I will change it anyway.